dp/cloud/python/magma/configuration_controller/response_processor/strategies/response_processing.py

```diff
@@ -146,16 +140,14 @@ def _create_channels(response: DBResponse, session: Session):
         return
     for ac in available_channels:
         frequency_range = ac["frequencyRange"]
-        channel = DBChannel(
-            cbsd=cbsd,
-            low_frequency=frequency_range["lowFrequency"],
-            high_frequency=frequency_range["highFrequency"],
-            channel_type=ac["channelType"],
-            rule_applied=ac["ruleApplied"],
-            max_eirp=ac.get("maxEirp"),
-        )
-        logger.info(f"Creating channel for {cbsd.id=}")
-        session.add(channel)
+        channel = {
+            "low_frequency": frequency_range["lowFrequency"],
+            "high_frequency": frequency_range["highFrequency"],
+            "channel_type": ac["channelType"],
```

Comment:
I think we can skip `channel_type` and `rule_applied`, since we are not using those fields at all